sdk/include/aditof/frame.h

```diff
@@ -136,6 +136,15 @@ class Frame {
     SDK_API virtual Status getAttribute(const std::string &attribute,
                                         std::string &value) const;
 
+    /**
+     * @brief Gets the temperature values stored in the metadata for each frame
+     * @param[out] sensorTemp - Sensor temperature
```

Comment:
Do you how the measurement unit for these temperatures? It would add more clarity so say something like:
"@param[out] sensorTemp - Sensor temperature in Celsius/Fahrenheight/Kelvin degrees.